ansibleflow/run.py

```diff
@@ -42,12 +45,20 @@ def build_ansible_command(playbook, target, environment):
     if target.tags:
         command += ' --tags "{0}"'.format(target.tags)
 
+    if arguments.extra_vars and len(arguments.extra_vars) > 0:
+        extra = arguments.extra_vars[0]
+        if extra.startswith('"') or extra.startswith("'"):
+            command += ' --extra-vars {0}'.format(extra)
+        else:
+            command += ' --extra-vars "{0}"'.format(extra)
+
     return command
 
 
 def run(target_name, env_name, arguments, dry_run=False):
-    target = get_config().targets.get(target_name, None)
-    environment = get_config().environments.get(env_name, None)
+    target = get_config(filename=arguments.file).targets.get(target_name)
+    environment = get_config(filename=arguments.file).environments.get(
+            env_name)
```

Comment:
This is a rather ugly line wrap. It would probably be better to just assign the config to a local var so that you don't have to use the line wrap at all.